web/src/components/circulargauge.js

```diff
@@ -6,6 +6,12 @@ import { arc } from 'd3-shape';
 /* eslint-disable jsx-a11y/mouse-events-have-key-events */
 // TODO: re-enable rule
 
+/*
```

Comment:
a note here related to another bug that I saw sometimes:

Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
    in Motion
    in g
    in g
    in svg
    in div
    in Unknown (created by TooltipContent)
    in div (created by TooltipContent)
    in div (created by TooltipContent)
    in div (created by TooltipContent)
    in div (created by TooltipContent)
    in TooltipContent (created by MapCountryTooltip)
    in div (created by Tooltip)
    in Portal (created by Tooltip)
    in Tooltip (created by MapCountryTooltip)
    in MapCountryTooltip (created by ConnectFunction)
    in ConnectFunction (created by CountryHistoryCarbonGraph)